synapse/rest/client/versions.py

```diff
@@ -44,10 +48,22 @@ def on_GET(self, request):
                     "r0.5.0",
                 ],
                 # as per MSC1497:
-                "unstable_features": {"m.lazy_load_members": True},
+                "unstable_features": {
+                    "m.lazy_load_members": True,
+                    # Advertise to clients whether they need not include an `id_server`
+                    # parameter during registration or password reset, as Synapse now decides
+                    # itself which identity server to use (or none at all).
+                    #
+                    # This is also used by a client when they wish to bind a 3PID to their
+                    # account, but not bind it to an identity server, the endpoint for which
+                    # also requires `id_server`. If the homeserver is handling 3PID
+                    # verification itself, there is no need to ask the user for `id_server` to
+                    # be supplied.
+                    "m.require_identity_server": self.config.account_threepid_delegate is None
```

Comment:
```suggestion
                    "m.require_identity_server": self.config.account_threepid_delegate is None,
```

Reply:
That's not how JSON works Dave!

Reply:
Oh but hey, this is a python dict, not JSON huh.